.github/workflows/workflow.yml

```diff
@@ -56,6 +56,8 @@ jobs:
           clickhouse: "22.8.21.38"
         - python: "3.10"
           clickhouse: "23.3.13.6"
+        - python: "3.10"
+          clickhouse: "23.8.3.48"
         - python: "3.10"
           clickhouse: "latest"
```

Comment:
Explicitly add the LTS version 23.8.

23.9 is `latest` right now. Don't add it explicitly as it has short support lifecycle.